packages/next/test/request.spec.ts

```diff
@@ -1,6 +1,6 @@
 import Chance from 'chance'
 import { NextURL } from 'next/dist/server/web/next-url'
-import { NextCookies } from 'next/dist/server/web/spec-extension/cookies'
+import { RequestCookies } from 'next/dist/server/web/spec-extension/cookies'
```

Comment:
Is this to fixe the `RespsonseCookies.getAll()`?

Reply:
@nickytonline No, this is related to changes in Next 13, where they switched from a single `NextCookies` class into separate `RequestCookies` and `ResponseCookies` classes